rust/helm/hyperlane-agent/templates/relayer-external-secret.yaml

```diff
@@ -35,7 +35,7 @@ spec:
   {{- if eq .signer.type "hexKey" }}
   - secretKey: {{ printf "%s_signer_key" .name }}
     remoteRef:
-      key: {{ printf "%s-%s-key-relayer" $.Values.hyperlane.context $.Values.hyperlane.runEnv }}
+      key: {{ printf "%s-%s-key-%s-relayer" $.Values.hyperlane.context $.Values.hyperlane.runEnv .name }}
```

Comment:
If we merge this change, we would break our infrastructure with it. Can you make this configurable to support both ways?

Reply:
@nambrot we are also gonna migrate from one signer key to multiple keys soon and not using this yet tbh.
But i think this migration just requires external secrets creation and no need any other rollout or changes in k8s applications.

Reply:
btw, for the backward compatibility, we can introduce a new flag value to switch secret key naming but it seems not a smart solution

Reply:
got it. I added a new flag value for backward compatibility. Please review again.